virttest/qemu_storage.py

```diff
@@ -1552,9 +1621,16 @@ def support_cmd(self, cmd):
         """
         supports_cmd = True
 
-        if cmd not in self.help_text:
-            LOG.error("%s does not support command '%s'", self.image_cmd, cmd)
-            supports_cmd = False
+        cmd_help_text = self._get_cmd_help_text(cmd)
```

Comment:
@YongxueHong  The `support_cmd` is used to check the sub command is supported by the `image_cmd`.
 Use`$image_cmd -h`  to get help_text is still valid in this function evenif it is for new qemu version.

For example, when checking if `compare` is supported by the $image_cmd,  the help_text is  `$image_cmd -h` all the time,  never use `$image_cmd compare -h`.

Reply:
Hi @yanan-fu 
Thanks, you point makes sense, I will update it later.